rocm-gdb/PKGBUILD

```diff
@@ -0,0 +1,54 @@
+# Maintainer Torsten Keßler <t dot kessler at posteo dot de>
+pkgname=rocm-gdb
+pkgver=3.5.0
+pkgrel=1
+pkgdesc='ROCm source-level debugger for Linux, based on GDB'
+arch=('x86_64')
+url='https://github.com/ROCm-Developer-Tools/ROCgdb'
+license=('custom: GPL with AMD copyright')
+depends=('rocm-dbgapi' 'python' 'guile2.0' 'ncurses' 'expat' 'xz' 'zlib' 'mpfr' 'source-highlight' 'babeltrace')
+makedepends=('texinfo')
+source=("$pkgname-$pkgver.tar.gz::$url/archive/rocm-$pkgver.tar.gz")
+sha256sums=('cf36d956e84c7a5711b71f281a44b0a9708e13e941d8fca0247d01567e7ee7d1')
+_dirname=$(basename "$url")-$(basename "${source[0]}" ".tar.gz")
+
+prepare() {
+    cd "$_dirname"
+    # From https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/gdb
+    # hack! - libiberty configure tests for header files using "$CPP $CPPFLAGS"
+    sed -i "/ac_cpp=/s/\$CPPFLAGS/\$CPPFLAGS -O2/" libiberty/configure
+}
+
+build() {
+    cd "$_dirname"
+    mkdir -p build && cd build
+    ../configure \
+    --prefix=/opt/rocm \
+    --program-prefix=roc \
+    --disable-shared \
+    --disable-nls \
+    --enable-source-highlight \
+    --enable-tui \
+    --enable-64-bit-bfd \
+    --enable-targets="$CHOST,amdgcn-amd-amdhsa" \
+    --with-system-readline \
+    --with-python=/usr/bin/python \
+    --with-guile=guile-2.0 \
+    --with-expat \
+    --with-system-zlib \
+    --with-babeltrace \
+    --with-lzma \
+    --disable-gdbtk \
+    --disable-ld \
+    --disable-gas \
+    --disable-gdbserver \
+    --disable-sim
```

Comment:
One level of indentation here would make it much better to read.

Reply:
You're right. I've added one more level of indentation.